examples/js/loaders/GLTFLoader.js

```diff
@@ -214,7 +221,17 @@ THREE.GLTFLoader = ( function () {
 
 			} );
 
-			parser.parse( onLoad, onError );
+			if ( this.lazy ) {
+
+				parser.markDefs();
```

Comment:
Could `markDefs` be moved into the parser's constructor? Then it would be called in one place for both lazy and non-lazy modes.